project1/application.py

```diff
@@ -51,25 +51,25 @@ def register():
 #     return "Hello "+name.split('@')[0]+"! You have successfully registered"
 
 
-@app.route("/display", methods=["POST"])
+@app.route("/display", methods=["POST", "GET"])
 def display():
-    uname = request.form.get("Email")
-    pwd = request.form.get("password")
-    tstamp = datetime.datetime.now()
-    print(uname, pwd, tstamp)
-    try:
-        db.execute("INSERT INTO userdata(username, passwords, creationstamp) VALUES (:username, :passwords, :creationstamp)",
-                   {"username": uname, "passwords": pwd, "creationstamp": tstamp})
-        db.commit()
-        return "Hello "+uname.split('@')[0]+"! You have successfully registered"
-    except:
-        return "Hello "+uname.split('@')[0] + "! Failed to Register"
+    if request.method == "POST":
+        uname = request.form.get("Email")
+        pwd = request.form.get("password")
+        tstamp = datetime.datetime.now()
+        print(uname, pwd, tstamp)
+        try:
+            db.execute("INSERT INTO userdata(username, passwords, creationstamp) VALUES (:username, :passwords, :creationstamp)",
+                       {"username": uname, "passwords": pwd, "creationstamp": tstamp})
```

Comment:
Instead of SQL queries try using SQLAlchemy commands like add() function.

Reply:
Will modify and update it along with today's tasks.